### PR TITLE
docs: add Akeyless to supported providers

### PR DESCRIPTION
**Pull request by akljph**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

### CODE REVIEW

**Comment by akljph** [comment]
This would also include adding us to the following pages:

https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/358531e35b4df40bacf58e88d6bf7b5e8a2b1c42/docs/book/src/providers.md

https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/358531e35b4df40bacf58e88d6bf7b5e8a2b1c42/docs/book/src/introduction.md

https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/358531e35b4df40bacf58e88d6bf7b5e8a2b1c42/docs/book/src/concepts.md

And on a bunch of other pages in the docs (sometimes inside comments on referenced YAMLs, etc.)

**Comment by aramase** [comment]
/retitle docs: add Akeyless to supported providers

File: docs/book/src/getting-started/installation.md [hunk 1]
```diff
@@ -88,3 +88,4 @@ Now that the Secrets Store CSI Driver has been deployed, select a provider from
 - [Azure Provider](https://azure.github.io/secrets-store-csi-driver-provider-azure/)
 - [GCP Provider](https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp)
 - [Vault Provider](https://github.com/hashicorp/secrets-store-csi-driver-provider-vault)
+- [Akeyless Provider](https://github.com/akeylesslabs/akeyless-csi-provider)
```
Comment (aramase) [inline docs/book/src/getting-started/installation.md:91]: The current list of providers is alphabetical, so move this to the top of the list.

**Review by aramase — CHANGES_REQUESTED** [review]
Thanks for the PR! 

@akljph Are you planning to address https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/1306#issuecomment-1647569727 in this PR?

**Comment by akljph** [comment]
> Thanks for the PR!
> 
> @akljph Are you planning to address [#1306 (comment)](https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/1306#issuecomment-1647569727) in this PR?

I just wasn't sure what the procedure was. Should I go through and find every possible edit and add it to the PR? Or create a new PR with this and all other edits?

File: docs/book/src/getting-started/installation.md (hunk 1 above)
Comment (akljph) [reply docs/book/src/getting-started/installation.md]: Done in latest commit.

**Comment by akljph** [comment]
Hi @aramase and @ritazh - any updates here? Thanks!

**Comment by akljph** [comment]
Hi @aramase and @ritazh - any updates here? Thanks!

**Review by aramase — COMMENTED** [review]
/ok-to-test

**Review by aramase — COMMENTED** [review]
Thank you for the PR! 🎉 

/label tide/merge-method-squash

/lgtm
/approve

**Pull request merged**